src/Transports/MassTransit.KafkaIntegration/KafkaIntegration/ConsumerLockContext.cs

```diff
@@ -56,6 +57,20 @@ public Task Faulted(ConsumeResult<byte[], byte[]> result, Exception exception)
             return Task.CompletedTask;
         }
 
+        public async ValueTask DisposeAsync()
+        {
+        }
```

Comment:
```suggestion
        public ValueTask DisposeAsync()
        {
            return ValueTask.CompletedTask;
        }
```

Reply:
`ValueTask.CompletedTask` is part of .NET5 +, but I can use default tho